cli/popper/runner_host.py

```diff
@@ -157,14 +103,62 @@ def run(self, step):
             log.step_info(pu.decode(line).strip('\n'))
 
         e = container.wait()['StatusCode']
-
         return e
 
     def stop_running_tasks(self):
         for c in DockerRunner.spawned_containers:
             log.info(f'Stopping container {c.name}')
             c.stop()
 
+    @staticmethod
+    def prepare_image(step, config):
```

Comment:
excellent. This refactoring is going to be very useful to reuse on runners for other platforms

Reply:
Yeah